www/MobileServices.Web.js

```diff
@@ -10673,6 +10677,16 @@
 		                }
 		                return this._gcm;
 		            }
+		        },
+		        'wns': {
+		            get: function () {
+		                if (!this._wns) {
+		                    var name = _.format('MS-PushContainer-gcm-{0}', mobileServicesClient.applicationUrl);
```

Comment:
Here not would be 'MS-PushContainer-wns-{0}' instead of 'MS-PushContainer-gcm-{0}' ?